app/javascript/stylesheets/application.scss

```diff
@@ -0,0 +1,26 @@
+@import "~bootstrap/scss/bootstrap";
+
+.height-20 {
```

Comment:
start creating a folder/file structure for styles
examples:
base/_variables
base/_mixins
base/_elements
base/_index
films/_show